### PR TITLE
Fix spells

### PR DESCRIPTION
**Pull request by MohamadKh75**
❤️

### CODE REVIEW

**Comment by MohamadKh75** [comment]
ping @maritome

File: src/allureReport.js [hunk 1]
```diff
@@ -61,26 +61,27 @@ async function readAllureReport(reportPath, testEnvPath) {
 		(testRes) => testRes.status === Status.SKIPPED
 	)
 	if (failedTests.length > 0) {
-		title = `${failedTests.length} test case(s) failed`
+		title = `${failedTests.length} test ${
+			failedTests.length === 1 ? 'case is' : 'cases are'
+		} failed`
```
Comment (maritome) [inline src/allureReport.js:66]: I would prefer `title = `${failedTests.length} test ${failedTests.length === 1 ? 'case' : 'cases'} failed``

Comment (MohamadKh75) [reply src/allureReport.js:66]: Fixed

**Comment by MohamadKh75** [comment]
Updated @maritome

**Comment by MohamadKh75** [comment]
ping @maritome

**Pull request merged**